llvm/lib/Target/EraVM/EraVMTieSelectOperands.cpp

```diff
@@ -58,55 +58,35 @@ char EraVMTieSelectOperands::ID = 0;
 INITIALIZE_PASS(EraVMTieSelectOperands, DEBUG_TYPE,
                 ERAVM_TIE_SELECT_OPERANDS_NAME, false, false)
 
-/// try to emit a new select instruction which contains an implicit tie
-/// so that the register allocator can coalesce both registers.
-/// Return true if a new instruction was emitted.
+/// Try to create an implicit tie so that the register allocator can coalesce
+/// both registers.
+/// Return true if we managed to do so.
 bool EraVMTieSelectOperands::tryPlacingTie(MachineInstr &MI,
                                            EraVM::ArgumentKind Arg) const {
   assert(Arg == EraVM::ArgumentKind::In0 || Arg == EraVM::ArgumentKind::In1);
-  // Given that we've already made constraints on the case of single
```

Comment:
We still handle immediate (not CP) case in td, so the comment is to be updated, not removed.

Reply:
Added comment to the newly introduced function.